sdk/core/Azure.Core/src/Pipeline/HttpPipeline.cs

```diff
@@ -66,11 +66,14 @@ public HttpMessage CreateMessage()
         /// <param name="message">The <see cref="HttpMessage"/> to send.</param>
         /// <param name="cancellationToken">The <see cref="CancellationToken"/> to use.</param>
         /// <returns>The <see cref="ValueTask"/> representing the asynchronous operation.</returns>
-        public ValueTask SendAsync(HttpMessage message, CancellationToken cancellationToken)
+        public async ValueTask SendAsync(HttpMessage message, CancellationToken cancellationToken)
         {
             message.CancellationToken = cancellationToken;
             AddHttpMessageProperties(message);
-            return _pipeline.Span[0].ProcessAsync(message, _pipeline.Slice(1));
+
+			// TODO: We need to discuss possible performance implications of this
+            await _pipeline.Span[0].ProcessAsync(message, _pipeline.Slice(1)).ConfigureAwait(false);
```

Comment:
Question: what are the performance implications of making this change?

Reply:
I would propose that we don't do this change now. After LLC RC, we can measure the impact and either do this change or leave the code as-is.

Reply:
That makes sense. Given that this approach relies on `Response.RequestClassifier` being internal, the approach to putting this functionality in Azure.Core.Experimental may end up looking quite different. I can do that implementation, but I wanted to make sure we discussed what it would look like in Core in the long term so we didn't end up with unpleasant surprises when we made that transition.